forge/src/Jackal.sol

```diff
@@ -4,13 +4,24 @@ pragma solidity ^0.8.0;
 import {Strings} from "@openzeppelin/contracts/utils/Strings.sol";
 
 abstract contract Jackal {
-    event PostedFile(address sender, string merkle, uint64 size);
+    event PostedFile(address sender, string merkle, uint64 size, string note);
```

Comment:
my bad we need to add an expires field to this, usually it will be left at 0 to just use the storage plan, but for posting a file for X years we'll need an expires field.